frontend/package.json

```diff
@@ -63,15 +63,15 @@
     "@nuxtjs/sitemap": "^7.0.0",
     "@nuxtjs/tailwindcss": "^6.12.1",
     "@pinia/nuxt": "^0.9.0",
-    "@sentry/node": "^8.26.0",
-    "@sentry/vue": "^8.26.0",
+    "@sentry/nuxt": "^8.45.0",
     "@tailwindcss/typography": "^0.5.13",
     "@vueuse/core": "^12.0.0",
     "@wordpress/is-shallow-equal": "^5.3.0",
     "async-mutex": "^0.5.0",
     "axios": "^1.7.2",
     "axios-mock-adapter": "^1.22.0",
     "clipboard": "^2.0.11",
+    "dotenv": "^16.4.7",
```

Comment:
This surprised me because I had read that [Nuxt has `dotenv` built-in](https://nuxt.com/docs/guide/directory-structure/env).

Reply:
Right, we already had it in the lock file due to Nuxt. I added it because it was mentioned in the Nuxt Sentry module setup docs, but removed it after your comment.